x-pack/plugins/ml/public/application/components/jobs_awaiting_node_warning/new_job_awaiting_node_shared/new_job_awaiting_node_shared.tsx

```diff
@@ -107,7 +107,7 @@ const MLJobsAwaitingNodeWarning: FC<Props> = ({ jobIds }) => {
         <div>
           <FormattedMessage
             id="xpack.ml.jobsAwaitingNodeWarningShared.noMLNodesAvailableDescription"
-            defaultMessage="There {jobCount, plural, one {is} other {are}} {jobCount, plural, one {# job} other {# jobs}} waiting for machine learning nodes to start."
+            defaultMessage="There {jobCount, plural, one {is} other {are}} {jobCount, plural, one {# job} other {# jobs}} waiting for autoscaling to add ML capacity. This may take several minutes."
```

Comment:
Same as above.
```suggestion
            defaultMessage="{jobCount, plural, one {# job} other {# jobs}} will start after autoscaling has increased ML capacity. This may take several minutes."
```